examples/stack-full.yaml

```diff
@@ -1,20 +1,21 @@
+---
 stack:
-  repository: "https://github.com/gimlet-io/gimlet-stack-reference.git"
+  repository: https://github.com/tcrst/gimlet-stack-reference.git?tag=v0.6.0
 config:
-  nginx:
-    enabled: true
-    host: mycompany.com
+  certManager:
+    email: me@mycompany.com
   loki:
     enabled: true
-  prometheus:
+  nginx:
     enabled: true
-  certManager:
-    email: "me@mycompany.com"
+    host: mycompany.com
   oauth2Proxy:
+    clientID: xxx
+    clientSecret: xxx
+    enabled: false
+    org: My Company
+    user: githubUser
+  prometheus:
     enabled: true
-    org: "My Company"
-    user: "githubUser"
-    clientID: "xxx"
-    clientSecret: "xxx"
   sealedSecrets:
-    sealingKey: "xxx"
+    sealingKey: xxx
```

Comment:
I could be wrong, but yaml strings are safest to be quoted I think.

Reply:
Reverted to the initial one